hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -0,0 +1,285 @@
+import {
+    Button,
+    Checkbox,
+    FormControlLabel,
+    Grid,
+    Link,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import { mockPendingOrders, mockHardware } from "testing/mockData";
+import React from "react";
+import Container from "@material-ui/core/Container";
+import styles from "components/general/OrderTables/OrderTables.module.scss";
+import hardwareImagePlaceholder from "assets/images/placeholders/no-hardware-image.svg";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import FormControl from "@material-ui/core/FormControl";
+import {
+    GeneralOrderTableTitle,
+    GeneralOrderTitle,
+} from "components/general/OrderTables/OrderTables";
+import { Formik, Field } from "formik";
+
+const createDropdownList = (number: number) => {
+    let entry = [];
+
+    for (let i = 0; i <= number; i++) {
+        entry.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+
+    return entry;
+};
+
+const setInitialValues = (request: { id: number; requested_quantity: number }[]) => {
+    let orderInitalValues: Record<string, string | boolean> = {};
+    request.forEach((orderItem) => {
+        orderInitalValues[`${orderItem.id}-quantity`] =
+            orderItem.requested_quantity.toString();
+        orderInitalValues[`${orderItem.id}-checkbox`] = false;
+    });
+    return orderInitalValues;
+};
+
+const convertToDateTime = (dateString: string) => {
+    const date = new Date(dateString);
+    const formattedDateString =
+        ("0" + date.getUTCHours()).slice(-2) +
```

Comment:
reminder to change this to Local time!
you can use date.toLocaleString() or a combination of date.toLocaleDate() and date.toLocaleTime()